index.js

```diff
@@ -256,9 +256,10 @@ function wordSelectionStart(text: string, i: number): number {
   return index
 }
 
-function wordSelectionEnd(text: string, i: number): number {
+function wordSelectionEnd(text: string, i: number, multiline: boolean): number {
   let index = i
-  while (text[index] && !text[index].match(/\s/)) {
+  const breakPoint = multiline ? /\n/ : /\s/
+  while (text[index] && !text[index].match(breakPoint)) {
```

Comment:
This is the fix. Previously when `selectionStart === selectionEnd`, text selection is expanded to the first occurrence of `space`. This changes it so selection is expanded to the first line break when appropriate (`ul`/`blockquote`/`task-lists`).